### PR TITLE
Update rubygem-fog-google to 1.15.0, rubygem-google-api-client to 0.50.0

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/foreman/rubygem-fog-google/rubygem-fog-google.spec [hunk 1]
```diff
@@ -110,6 +108,9 @@ cp -pa .%{gem_dir}/* \
 %doc %{gem_instdir}/MIGRATING.md
 
 %changelog
+* Thu May 19 2022 Evgeni Golov 1.15.0-1
```
Comment (evgeni) [inline packages/foreman/rubygem-fog-google/rubygem-fog-google.spec:111]: Not updating to 1.16+ yet, as that needs a whole bunch new deps.

**Comment by evgeni** [comment]
needs to be merged once https://github.com/theforeman/foreman/pull/9226 is in

**Review by ehelms — APPROVED** [review]

**Comment by ehelms** [comment]
That current EL7 failure looks unpleasant.

**Comment by evgeni** [comment]
> That current EL7 failure looks unpleasant.

rexml is a built-in gem before 3.0, so for now we can just drop it ;)

**Pull request merged**